src/ptvsd/common/launcher.py

```diff
@@ -78,5 +80,21 @@ def parse(argv):
     return it
 
 
+def _send_pid(pid):
+    if "PTVSD_PID_SERVER_PORT" not in os.environ:
+        return
+
+    from ptvsd.common import socket
+    port = int(os.environ["PTVSD_PID_SERVER_PORT"])
+    with socket.create_client() as sock:
+        sock.connect(("127.0.0.1", port))
+        sock.sendall(str(pid).encode("ascii"))
```

Comment:
I'm not entirely sure what this does on Python 2, since `str` is `bytes` there... does it even have `encode()`? 

But there's a handy trick with `%` operator for getting a number represented as `bytes` on any Python version consistently:
```py
b"%d" % pid
```
(In Python 3, `bytes` doesn't have `format()`, but it still supports `%`)